MovieApp/MovieApp/App/Features/MovieSearch/MoviesViewController/MoviesViewPresenter.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+import Alamofire
+
+class MoviesViewPresenter {
+
+    private let networkService: NetworkService
+    weak private var moviesViewDelegate: MoviesViewController?
+    
+    init(networkService: NetworkService){
+        self.networkService = networkService
+    }
+    
+    func setMoviesViewDelegate(moviesViewDelegate: MoviesViewController?){
+        self.moviesViewDelegate = moviesViewDelegate
+    }
+
+    func fetchMovies() {
+        networkService.fetchMovies { [weak self] (result: Result<Movies, ApiError>) in
```

Comment:
You are missing a specific client for this task. We don't use networkService throughout the app. We create clients that use our generic functions from networkService for specific tasks and implement that clientsProtocol inside (for now) presenters.  Having that in mind, naming fetchMovies in networkService is wrong.

Reply:
Will do in next pull request which will include more architectual changes if that's ok. I saved your comment locally on Mac so I don't forget about it.

Reply:
Fixed